### ISSUE TITLE
Request: Environment variable to turn of log collection

### ISSUE THREAD

**Issue by binaryseed** [newrelic-experimental/gha-new-relic-exporter#31]
Would be nice to be able to turn off log collection (ie: sending all the logs to New Relic)

**Comment by dpacheconr** [comment]
currently logs are used to set Span status, so we would still required to obtain the logs via Github API
https://github.com/newrelic-experimental/gha-new-relic-exporter/blob/main/src/exporter.py#L183
Guess we could request the logs, parse them, set the status and then based on variable decide to send them to New Relic or not
it will not have logs in context functionality in NR for sure, but will need to review if there is any further impact but I guess it's doable
do you want to completely disable log collection including obtain logs from github api or simply ability to stop sending them to NR and lose certain functionality like logs in context?

**Comment by binaryseed** [comment]
Yeah, the thought would be to simply avoid pushing many thousands of log lines up to NR, since we can just navigate to the action to see all the logs. The exporter is already valuable to us w/o incurring the ingest cost of all the log lines

**Comment by dpacheconr** [comment]
https://github.com/newrelic-experimental/gha-new-relic-exporter/releases/tag/1.0.0

**Comment by binaryseed** [comment]
Thanks!
